### ISSUE TITLE
[Frontend] Unnest medical care questions from schema on 10-10CG

### ISSUE THREAD

**Issue by ala-yna** [department-of-veterans-affairs/va.gov-team#15160]
## User Story
As a Veteran,
I have the option of entering where I last received medical care on the 10-10CG.
Because this is an optional two-part question, I want both parts to be required if one part is filled in, and otherwise, I want to be able to leave them both blank.

## Expected Behavior
Unnest the two properties so they are no longer dependent on each other. Currently due to them being in the same object they are both optionally required depending if the other value has data. Make it so they are indigent of each other.

## Acceptance Criteria

- [x] veteran previous medical facility questions are no longer dependent on each other

**Comment by LinnJS** [comment]
This is on hold due to breaking change on CARMA and wanting to group the changes.

**Comment by LinnJS** [comment]
[PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15030) with the FE work

**Comment by ala-yna** [comment]
Status: waiting to coordinate with CARMA to make sure this isn't a breaking change.

**Comment by mlighdc** [comment]
@ala-yna and @LinnJS have you coordinated with CARMA yet on this change? Where is the blocker?

**Comment by ala-yna** [comment]
We were waiting to batch this request with other changes, including the changes needed to support the new paper version of the form. I just followed up with Matt and he should be sending it over today.

**Comment by LinnJS** [comment]
[PR](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/549) for schema updates

**Comment by LinnJS** [comment]
Need to write email to program team once FE, BE, and Schema work is complete.

**Comment by mlighdc** [comment]
@ala-yna there seems to be several tickets with the same issue. Can you please consolidate?

**Comment by ala-yna** [comment]
There are two tickets related to this issue, one for frontend and one for backend. This is something we should include in CARMA's next PI planning cc @tchrisdyer

**Comment by hdjustice** [comment]
In discussion this with Lihan, changing how the attributes are formatted and sent is not worth the risk of breaking something in the backend.  There are no issues on how the data is sent currently, and we feel this change is simply not necessary, and not worth the risk.
Stakeholders were informed on 11/4/2022 in routine touchbase call
